02-cpu-configure/rffe-reset.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -eu
+if [ $# -ne 1 ]; then
+    echo "Usage: $0 <virtual_slot>"
+    exit 1
+fi
+
+expect << EOF
+spawn telnet 192.168.2.$(( 190 + $1 ))
+expect "NuttShell (NSH)"
+send "reboot\r"
```

Comment:
Why is the `\r` necessary?

Reply:
According to the [expect manual](https://linux.die.net/man/1/expect):

> send [-flags] string
>     Sends string to the current process. For example, the command
> 
>     send "hello world\r"
> 
>     sends the characters, h e l l o <blank> w o r l d <return> to the current process. (Tcl includes a printf-like command (called format) which can build arbitrarily complex strings.) 
> Characters are sent immediately although programs with line-buffered input
>     will not read the characters until a return character is sent. A return character is denoted "\r".